.github/workflows/ci-cd-backend.yml

```diff
@@ -0,0 +1,103 @@
+name: Build PackIt Backend
+
+on:
+  push:
+    branches:
+      - '**' 
+    tags-ignore: 
+      - 'v*'
+    paths:
+      - 'packit-backend/**'
+      - '.github/workflows/ci-cd-backend.yml'
```

Comment:
Please can you add the checks for the PR?

Reply:
done already